backend/gccbackend/server/serializers.py

```diff
@@ -15,9 +15,63 @@ class Meta:
         fields = ['url', 'name']
 
 class LessonSerializer(serializers.ModelSerializer):
+    in_chapter = serializers.IntegerField(write_only=True)
+    get_chapter = serializers.IntegerField(read_only=True)
+    get_lesson = serializers.IntegerField(read_only=True)
+    
     class Meta:
         model = Lesson
-        fields = ['number', 'title', 'content', 'chapter']
+        fields = ['number', 'title', 'content', 'in_chapter', 'get_chapter', 'get_lesson']
+
+    def get(self, validated_data):
+        print('IN GET')
+        return Lesson.objects.filter(number=validated_data.pop('get_lesson'), chapter__number=validated_data.pop('get_chapter')).first()
+        lesson = Lesson.objects.filter(number=validated_data.pop('set_lesson'), chapter__number=validated_data.pop('set_chapter')).first()
```

Comment:
Should it return or assign to lesson? Code under the return is unreachable now.

Reply:
cleaned up all the comments, prints and unreachable code (the whole get is unreachable)